ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -23,6 +26,7 @@
 
 LOOKAHEAD_WPS = 200 # Number of waypoints we will publish. You can change this number via parameter
 PUBLISHER_RATE = 1  # Publishin rate on channel /final_waypoints
+MAX_SPEED = 10 # replace with the configurable one
```

Comment:
I think we can take it from base_waypoints to avoid duplication.
the waypoint_loader has this parameter in launch file. and the base_waypoints are published once and never change